templates/zerver/help/name-change.md

```diff
@@ -0,0 +1,16 @@
+#Change your name
+
+![Settings](/static/images/help/settings.png)
+
+If for whatever reason you ever wanted to change your Zuli username, name changes are supported. 
+
+Visit your settings either by clicking the cog widget on the top right corner and then settings or by redirecting your browser to  https://chat.zulip.org/#settings.
+
+Change your name on the Full Name field, and click save changes. 
+
+![Change your name](/static/images/help/name_change.png)
+
+Congratulations! You have a new identity 
+
+Note that currently there is a bug and you won't be able to see the changes in topics you already commented but only on new topics.
```

Comment:
This isn't quite correct -- posting in the same topic after someone else posts in it will show the new name. It's more accurate to say something like:

> Note that messages you've already sent will show the name you had when you sent them.